ddsc/core/util.py

```diff
@@ -302,7 +302,7 @@ def verify_terminal_encoding(encoding):
     :param encoding: str: encoding we want to check
     """
     encoding = encoding or ''
-    if not ("UTF" in encoding.upper()):
+    if not ("UTF" in encoding.upper()) and encoding:
```

Comment:
This is getting hard to follow. Might I suggest we clarify by removing the `encoding = encoding or ''` and write the conditional as: `if encoding and 'UTF' not in encoding.upper():`.